cmd/agent/app/start.go

```diff
@@ -126,6 +126,10 @@ func StartAgent() error {
 	if err != nil {
 		return fmt.Errorf("unable to set up global agent configuration: %v", err)
 	}
+	err = common.SetupConfigOSSpecifics()
```

Comment:
I think this should called/part of `common.SetupConfig` so that individual commands (e.g. the `check` command) don't have to call it separately. Or is there a reason not to?